src/components/Disputes/DisputeRoundPill.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { numberToWord } from '../../lib/math-utils'
+
+function DisputeRoundPill({ roundId }) {
+  if (roundId === undefined) return null
+
+  const label = `Round ${numberToWord(roundId)}`
+  return (
+    <span
+      css={`
+        padding: 1px 16px;
+        border-radius: 100px;
+        background: linear-gradient(
+          13.81deg,
+          rgba(255, 179, 109, 0.3) -0.55%,
+          rgba(255, 136, 136, 0.3) 88.44%
+        );
+        text-transform: uppercase;
+        font-size: 12px;
```

Comment:
Should we use [`label2`](https://github.com/aragon/aragon-ui/blob/master/src/style/text-styles.js#L50-L54) here?